src/tlapm_args.ml

```diff
@@ -59,6 +59,7 @@ let parse_args args opts mods usage_fmt =
     exit 2
 
 let show_where () =
+  (* TODO: `opam exec -- tlapm --where` produces invalid output. *)
```

Comment:
Is it still the case?

Reply:
Commit 47afb6b6c0fbfc297b9ffdbf8d73f943ddc11a99 resolves this.